eng/pipelines/typespec-ci.yml

```diff
@@ -43,6 +44,11 @@ jobs:
       Folder: specification/contosowidgetmanager/Contoso.WidgetManager
       DisplayName: Contoso.WidgetManager
 
+  - template: templates/steps/typespec-ci.yml
```

Comment:
@harunrst: This change adds your new spec to our CI validation pipeline.  If you rename or move the folder please update it here.